### PR TITLE
feat:implemented function

### PR DESCRIPTION
**Pull request by jayshreejaiswal**
- i have implemented greet function

### CODE REVIEW

File: math.js [hunk 1]
```diff
@@ -38,4 +38,8 @@ function mul(a,b){
  */
 function sub(a,b){
     return a-b;
+}
+
+function greet (){
+    return"good morning guys"
```
Comment (Padiprasannakumar123) [inline math.js]: mention name

Comment (Padiprasannakumar123) [inline math.js]: mention nAME

File: math.js [hunk 2]
```diff
@@ -38,4 +38,8 @@ function mul(a,b){
  */
 function sub(a,b){
     return a-b;
+}
+
+function greet (){
+    return"good morning vasudha"
```
Comment (Padiprasannakumar123) [inline math.js]: GIVE FULL NAME

**Review by Padiprasannakumar123 — CHANGES_REQUESTED** [review]

File: math.js (hunk 2 above)
Comment (jayshreejaiswal) [reply math.js]: done

**Review by Padiprasannakumar123 — COMMENTED** [review]
LGTM

**Comment by Padiprasannakumar123** [comment]
LGTM

**Pull request merged**